### ISSUE TITLE
Improve popup pagination indicator for accessibility

### ISSUE THREAD

**Issue by kirsty-hames** [adaptlearning/adapt-contrib-hotgraphic#245]
The current popup pagination uses a forward slash to indicate which item you're on e.g. 2 / 4

A screen reader announces this as '2 slash 4' which isn't that intuitive. 

I'd suggest updating the [schema default](https://github.com/adaptlearning/adapt-contrib-hotgraphic/blob/a34c6b7973d8015a588faae080c9baaf0b438ec6/properties.schema#L18) to use 'of' instead of the slash. I think this is more intuitive for both the on screen text and screen readers.

e.g. `"default": "{{itemNumber}} of {{totalItems}}",`

Please +1 this ticket if you agree to the suggested change.

**Comment by swashbuck** [comment]
+1

**Comment by StuartNicholls** [comment]
Yes, I suggested this in #244. @joe-allen-89 has updated both schemas to reflect this
